app/src/main/java/com/stevesoltys/seedvault/restore/install/ApkSplitCompatibilityChecker.kt

```diff
@@ -0,0 +1,118 @@
+package com.stevesoltys.seedvault.restore.install
+
+import android.content.Context
+import android.os.Build
+import android.util.Log
+
+private const val TAG = "SplitCompatChecker"
+
+private const val CONFIG_PREFIX = "config."
+private const val CONFIG_LENGTH = CONFIG_PREFIX.length
+
+private const val LDPI_VALUE = 120
```

Comment:
You should add a link to the source here, https://github.com/google/bundletool/blob/1.2.0/src/main/java/com/android/tools/build/bundletool/model/utils/ResourcesUtils.java#L72

Reply:
Even if I took those numbers from https://developer.android.com/training/multiscreen/screendensities#TaskProvideAltBmp ?

Reply:
Yes. :trollface: